src/bigint.rs

```diff
@@ -874,6 +876,75 @@ impl BigUint {
 
     /// Creates and initializes a `BigUint`.
     ///
+    /// The bytes are in big-endian byte order
```

Comment:
This (and a few other sentences below) should have a trailing `.`. Other than that, I think it's ready.